src/bakery_manager_mobile/lib/pages/home_page.dart

```diff
@@ -117,9 +130,9 @@ class _MyHomePageState extends State<MyHomePage> {
                   color: Color.fromARGB(255, 209, 125, 51),
                 ),
               ),
-              const Text(
-                'Owner',
-                style: TextStyle(
+              Text(
+                _roleName,
```

Comment:
to uppercase so it displays "Employee" and not "employee"

Reply:
Uppercased